doc/param.md

```diff
@@ -0,0 +1,40 @@
+@function can-route.param param
+@parent can-route.static
+
+@description Get a route path from given data.
+
+@signature `route.param(data)`
+
+  Parameterizes the raw JS object representation provided in data.
+
+  ```js
+  import {route} from "can";
+
+  const result = route.param( { type: "video", id: 5 } );
+  console.log( result ); // -> "&type=video&id=5"
+  ```
+  @codepen
+
+  @param {data} object The data to populate the route with.
+  @param {String} [currentRouteName] The current route name.  If provided, this can be used to "stick" the url to a previous route. By "stick", we mean that if there are multiple registered routes that match the `object`, the the `currentRouteName` will be used.
+
+  @return {String} The route, with the data populated in it.
+
+@body
+
+If a route matching the provided data is found, that URL is built
```

Comment:
In a `## Use` ?  Is there anything new here?  Maybe move this into the signature.

Reply:
Moved into signature, it showed remaining data being added as key/value parameters.